### PR TITLE
 allow single-queues

### PR DESCRIPTION
**Pull request by JoeZ99**
The default behavior for `events_manager` is to create one queue per consumer for a specific topic. Say `tasks_results` topic. If there are 5 consumers, then 5 queues will be created, and each time an event is broadcasted to `tasks_results` topic, it will show on those 5 queues.

This is to allow that only one queue per topic is created. There would be only one queue binded to `tasks_results` topic and all 5 consumers would consume from that queue. Therefore, each time an event is broadcasted to `task_results` we can be sure that **only one** of those 5 consumers actually consumes it.

### CODE REVIEW

File: lib/events_manager/application.ex [hunk 1]
```diff
@@ -10,14 +10,27 @@ defmodule EventsManager.Application do
...
+      Enum.map(
+        consumers,
+        fn
+          {topic, functions} ->
+            {EventsManager.Consumer,
+             [
+               connection_uri: connection_uri,
+               exchange_topic: topic,
+               consumer_functions: functions
+             ]}
```
Comment (agutierrezrodriguez) [inline lib/events_manager/application.ex:22]: I would remove this pattern matching. As you set in docs, the third option is mandatory in config.

**Review by agutierrezrodriguez — APPROVED** [review]

**Review by mjosecuchillo — APPROVED** [review]

File: lib/events_manager/application.ex (hunk 1 above)
Comment (JoeZ99) [reply lib/events_manager/application.ex:22]: @agutierrezrodriguez 
precisely, in docs I say it's an "alternate" so the map is still understandable. I did that only to be back-compatible. If you think I'm being too cautious, I'm happy to remove it (and change the README.md accordingly)

btw, what kind of version change do you think this PR deserves? major (`2.2.0` -> '3.0.0`) or minor (`2.2.0` -> `2.3.0`)

Comment (agutierrezrodriguez) [reply lib/events_manager/application.ex:22]: Major if break compatibility

Comment (JoeZ99) [reply lib/events_manager/application.ex:22]: @agutierrezrodriguez 
OK, I infer from your rather short response:
  - you think we should break compatibility
  - therefore, `2.2.0` -> `3.0.0`

Comment (agutierrezrodriguez) [reply lib/events_manager/application.ex:22]: yes

**Pull request merged**